cocos/physics/physx/physx-shared-body.ts

```diff
@@ -358,7 +358,7 @@ export class PhysXSharedBody {
     }
 
     setMask (v: number): void {
-        if (v === -1) v = 0xffffffff;
+        //if (v === -1) v = 0xffffffff;
```

Comment:
this might cause some unknown problems.

---
Note: for historical reasons, the Enum function has special treatment for -1. If you are not familiar with it, do not define a property with a value of -1.

https://docs.cocos.com/creator/manual/zh/physics/physics-group-mask.html